src/index.html

```diff
@@ -11,25 +11,49 @@
   <body>
 
     <header class="header">
-      <div class="container">
-        <a href="index.html"> <img class="logo-img"
+
+      <a class="a" href="index.html"> <img class="logo-img"
           src="./images/logo.png" alt="logo">
-        </a>
-        <nav class="nav">
-          <ul class="ul-decor">
-            <li class="li-set"> <a class="is-active nav-set" href="#apple">APPLE</a></li>
-            <li class="li-set"> <a class="nav-set" href="#samsung">SAMSUNG</a></li>
-            <li class="li-set"> <a class="nav-set" href="#smart">SMARTPHONES</a></li>
-            <li class="li-set"> <a class="nav-set" data-qa="hover"
-              href="#lap&comp">LAPTOPS & COMPUTERS</a>
-            </li>
-            <li class="li-set"> <a class="nav-set" href="#gadg">GADGETS</a></li>
-            <li class="li-set"> <a class="nav-set" href="#tabl">TABLETS</a></li>
-            <li class="li-set"> <a class="nav-set" href="#photo">PHOTO</a></li>
-            <li class="li-set"> <a class="nav-set last" href="#video">VIDEO</a></li>
-          </ul>
-        </nav>
-      </div>
+      </a>
+
+      <nav class="nav">
+        <ul class="ul-decor">
+          <li class="li-set">
+            <a class="is-active nav-set a" href="#apple">apple</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" href="#samsung">samsung</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" href="#smartphones">smartphones</a>
+          </li>
+
+          <li class="li-set">
+            <a class="nav-set a" data-qa="hover"
+              href="#laptops&computers">laptops & computers
```

Comment:
If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

![image](https://github.com/mate-academy/layout_moyo-header/assets/96138503/fd9bde5e-c0cf-490f-b43e-5dbd70ecb932)